code/addons/links/template/stories/scroll.stories.ts

```diff
@@ -1,41 +0,0 @@
-import { global as globalThis } from '@storybook/global';
-import { withLinks } from '@storybook/addon-links';
-
-export default {
-  component: globalThis.Components.Html,
-  parameters: {
-    chromatic: { disable: true },
-  },
-  decorators: [withLinks],
-};
-
-export const Basic = {
```

Comment:
Should you keep one test for scrolling?

Reply:
I'm not sure what it even tests? I think maybe in 6.5 it would test the Docs tab, that it scrolled when navigating within it - which doesn't seem relevant anymore.
Or is there anything else I'm missing?

Reply:
I thought it was so that there would be a directed focus to the link, even if it wasn't visible at first. There's a typo in the code here, it should be `margin-bottom` instead of `marginBottom`, but I tested and it does nothing.